public/resumeData.json

```diff
@@ -2,60 +2,29 @@
   "main": {
     "name":"The Blaze Coders",
     "image":"",
-    "bio":"Bio",
-    "contactmessage":"Here is where you should write your message to readers to have them get in contact with you.",
-    "social":[
-      {
-        "name":"facebook",
-        "url":"",
-        "className":"fa fa-facebook"
-      },
-      {
-        "name":"twitter",
-        "url":"",
-        "className":"fa fa-twitter"
-      },
-      {
-        "name":"google-plus",
-        "url":"",
-        "className":"fa fa-google-plus"
-      },
-      {
-        "name":"linkedin",
-        "url":"",
-        "className":"fa fa-linkedin"
-      },
-      {
-        "name":"instagram",
-        "url":"",
-        "className":"fa fa-instagram"
-      },
-      {
-        "name":"github",
-        "url":"",
-        "className":"fa fa-github"
-      },
-      {
-        "name":"skype",
-        "url":"",
-        "className":"fa fa-skype"
-      }
-    ]
+    "bio":"",
+    "description" : "Innovation, Team Work, Wonders"
   },
   "project":{
     "projects": [
       {
         "title":"Smart Car Parking",
         "category":"Smartest Car Parking On Earth",
         "image":"smartcar.png",
-        "url":"https://theblazecoders.github.io/smart-car-parking"
+        "url":"https://github.com/theblazecoders/smart-car-parking"
+      },
+      {
+        "title":"Smart Car Parking App",
+        "category":"A Flutter app for the smart car parking project",
+        "image":"flutter.jpg",
```

Comment:
Can you please post the flutter.jpg img in a comment?

Reply:
why???

Reply:
I just wanted to see what it is

Reply:
But the ajax is imporant